.travis.yml

```diff
@@ -149,7 +149,8 @@ matrix:
       script:
         - if [ $RAY_CI_LINUX_WHEELS_AFFECTED != "1" ]; then exit; fi
 
-        - ./ci/travis/test-wheels.sh
+        # Explicitly sleep 60 seconds for logs to go through
+        - ./ci/travis/test-wheels.sh || cat /tmp/ray/session_latest/logs/* || sleep 60 || false
```

Comment:
@simon-mo @rkooo567 can you explain these lines to me, what is this for?

Reply:
- `sleep 60`: Travis doesn't print out the full traceback sometimes. When the test fails, the machine shutdown and the log get cut off. 
- `cat logs`: If the test errors, we need to see the ray logs to debug. This is because we cannot reproduce the issue locally nor can we do it via ssh into travis.